### DISCUSSION TITLE
No julia framework ? why not test julia Genie...

### DISCUSSION THREAD

**Discussion by ilrico** [TechEmpower/FrameworkBenchmarks#6871] [General]
Not familiar with julia and wondering how it would rank. Genie seems pretty mature, and julia advertises speed. So... I guess it would be nice to see how it performs relatively.

my 5 cts

**Comment by NateBrady23** [comment]
We do have julia frameworks: https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/Julia

Edit: But I've also added it here: https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Suggested-Frameworks-Languages-and-Features
